server.js

```diff
@@ -4739,6 +4739,53 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// Issue Stats integration.
+camp.route(/^\/issuestats\/([^\/]+)\/(.+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
I'm not quite sure I follow.  The third path component currently selects between different badge types (Issue and PR).  Are you suggesting moving this to a single-letter parameter before the host, or something else?